Documentation/policy/language.rst

```diff
@@ -1020,54 +1015,6 @@ DNS Proxy (preferred)
 
         .. literalinclude:: ../../examples/policies/l7/dns/dns-visibility.json
 
-.. _DNS Polling:
```

Comment:
Could you also change `DNS Proxy (preferred)` above? That was in relation to polling so there isn't a preference now.

Reply:
Please address this one comment, I think we're good after that.

Reply:
Do you want me to remove the 'DNS Proxy ( preferred) section completely?